recipes/scyjava/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "scyjava" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6eb1218e763c4842a114adc5f5d945f7e728a06016e83b9c0e287784bda9eedc
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vvv"
```

Should we install with `--no-deps` flag?
```suggestion
  script: "{{ PYTHON }} -m pip install . -vvv  --no-deps"
```